components/board/fake/board.go

```diff
@@ -235,7 +235,7 @@ func (b *Board) StreamTicks(ctx context.Context, interrupts []board.DigitalInter
 	extra map[string]interface{},
 ) error {
 	for _, i := range interrupts {
-		i.AddCallback(ch)
+		AddCallback(i.(*DigitalInterruptWrapper), ch)
```

Comment:
Are all implementations of AddCallbak basically using pinwrappes or the pi version of them?
If so, we can make that function just take in a GigitalInterruptWrapper instead of a board.DigitalInterrupt. Will save us from casting.

Reply:
Not for this pr, fo rhtis pr let's just make sure the hardware works.